contents/blog/clickhouse-vs-snowflake.md

```diff
@@ -21,105 +21,111 @@ If I was to compare the databases to boats, ClickHouse is a jet ski – ultra-fa
 
-### Who uses Snowflake and ClickHouse?
+- **Virtual warehouses instead of specialized engines:** Unlike ClickHouse's specialized engines, which utilize hardware to optimize query execution, Snowflake segments its database into virtual warehouses sized from small to large. These warehouses can have larger or lesser compute to handle complex to easy queries and isolate workloads. However, while clever and organized, this approach is computationally and financially expensive when taken full advantage of.
 
-Part of Snowflake’s value prop is the broad appeal to both technical and non-technical users. Snowflake’s marketplace makes connecting business intelligence tools with warehouse data easy, which appeals to teams wanting explore their data. ClickHouse Cloud has a competitive offering and the ecosystem is growing fast, but still smaller than Snowflake’s.
+- **Search optimization service:** Introduced in 2021, Snowflake offers a search optimization service at its higher enterprise tiers. Because Snowflake is a closed-source product, not much is known about how the search optimization service works, but it can offer enormous performance gains (4x-100x) relative to un-optimized search for some types of queries. Unfortunately, while Snowflake likely uses similar mechanisms to ClickHouse, direct side-by-side analysis isn’t possible due to Snowflake’s closed-search tier. By extension, the search optimization service’s performance boost is unpredictable.
 
-**Snowflake customers** include AT&T, Jetblue, Anthem, and Capital One. Snowflake customers tend to be enterprise-level and have to deal with a lot of data. 
+Overall, ClickHouse's close-metal optimizations enable it to return aggregate values over a thousand times faster (and cheaper) than Snowflake. However, Snowflake’s approach is considerably more user-friendly as it requires less SQL-level optimization to take advantage of these organizational features.
 
-**ClickHouse’s customers** include Github, YouTube, Twitter, Sentry, Uber, CloudFlare, and Slack. We, at PostHog, also [use ClickHouse](/docs/how-posthog-works/clickhouse). It is core to our product and powers most of everything you see when you visit our [app](https://app.posthog.com). 
+### Who uses Snowflake and ClickHouse?
 
+Part of Snowflake’s value prop is the broad appeal to both technical and non-technical users. Snowflake’s marketplace makes connecting business intelligence tools with warehouse data easy, which appeals to teams wanting explore their data. ClickHouse Cloud has a competitive offering, but the current integrations are limited to a select few.
 
-### How is ClickHouse and Snowflake used?
+While a managed ClickHouse solution is similar to Snowflake’s integration war chest, it has a long way to go before it is competitive. It does, however, level the playing field considerably. Before ClickHouse Cloud, the on-prem only option made it significantly harder to scale compared to Snowflake’s one-click upgrades.
```

Comment:
```suggestion
ClickHouse and Snowflake are both used for analytics, but ClickHouse is more focused on analytical queries. ClickHouse is a great fit for PostHog because we are an analytics platform – we need to be able to answer questions like “what is the average time to convert for users who saw a certain feature?” or “what is the average time to convert for users who saw a certain feature?” in *real time* and *at speed of click*.
```